core/modules/system/system.install

```diff
@@ -98,6 +100,21 @@ function system_requirements($phase) {
     $requirements['php_register_globals']['value'] = $t('Disabled');
   }
 
+  // Test PHP max_input_vars setting.
+  $max_input_vars = ini_get('max_input_vars');
+  $requirements['php_max_input_vars'] = array(
+    'title' => $t('PHP form input max'),
```

Comment:
```suggestion
    'title' => $t('PHP form input limit'),
```

Reply:
I don't really like either of these. It's not a limit on forms, or even a limit on input. It's an input on items per form. How do we convey that?

Reply:
I tried `PHP form value limit` let's try that?

Reply:
Perhaps `Form input PHP limit`?